### DISCUSSION TITLE
When I “make pythonAPI” ,there is something wrong with

### DISCUSSION THREAD

**Discussion by MoolixLin** [carla-simulator/carla#5879] [Q&A]
When I download the carla and "make pythonAPI", There is no dist folder in PythonAPI/carla despite a successful output message. And I konw there something wrong when I make PythonAPI, I found the error like this and I search "https://downloads.apache.org/xerces/c/3/sources/xerces-c-3.2.3.zip",the website is NOT found 404, and then I search "https://downloads.apache.org/xerces/c/3/sources" and I couldn't find the file like "xerces-c-3.2.3.zip". Is the file update? How to download this flie "xerces-c-3.2.3.zip" in the commend "make pythonAPI".
![1](https://user-images.githubusercontent.com/99062044/198201654-14206ebd-2c3f-432b-8db0-2d88eadaf487.png)

**Comment by mgmike** [comment]
Try changing line 432 of Util/BuildTools/Setup.sh to https://archive.apache.org/dist/xerces/c/3/sources/xerces-c-3.2.3.tar.gz

**Reply by MoolixLin** [reply]
Thanks for you answer, I have solved my problem. But there is another problem, I don't know how to solve it. The error like this picture, the error is about xerces_3_2，“An external command that cannot be resolved”=“无法解析的外部命令”. Maybe my UE4 have something wrong, because I can't compile it success.
![image](https://user-images.githubusercontent.com/99062044/199047353-346b2408-3f55-4dd0-867d-733c2c2932a3.png)
